Backend/dev/migrations/check_migrations.sh

```
@@ -0,0 +1,448 @@
+    [ "$name" = "extension" ] && continue
+    ln -s "$d" "$INST/share/postgresql/$name"
+  done
+  cp -P "$PG_PREFIX"/share/postgresql/extension/* "$INST/share/postgresql/extension"/
+  cp -P "$POSTGIS"/share/postgresql/extension/* "$INST/share/postgresql/extension"/
+  cp -P "$PG_LIB_PREFIX"/lib/* "$INST/lib"/ 2>/dev/null || true
+  cp -P "$POSTGIS"/lib/* "$INST/lib"/
+
+  # Postgres only consults dynamic_library_path when the module path does not
+  # start with $libdir/. Postgis ships .control + .sql files referencing
+  # $libdir/postgis-3 etc. — strip that prefix so our $INST/lib dir is searched.
+  ( cd "$INST/share/postgresql/extension"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)